config.json

```diff
@@ -11,404 +11,404 @@
       "topics": [
       ]
     },
     {
-      "slug": "all-your-base",
-      "difficulty": 1,
+      "slug": "allergies",
+      "difficulty": 4,
       "topics": [
       ]
     },
     {
-      "slug": "bank-account",
-      "difficulty": 1,
+      "slug": "all-your-base",
+      "difficulty": 4,
       "topics": [
       ]
     },
     {
-      "slug": "crypto-square",
-      "difficulty": 1,
+      "slug": "largest-series-product",
+      "difficulty": 4,
       "topics": [
       ]
     },
     {
-      "slug": "kindergarten-garden",
-      "difficulty": 1,
+      "slug": "clock",
+      "difficulty": 4,
       "topics": [
       ]
     },
     {
-      "slug": "robot-simulator",
-      "difficulty": 1,
+      "slug": "matrix",
+      "difficulty": 4,
       "topics": [
       ]
     },
     {
-      "slug": "queen-attack",
-      "difficulty": 1,
+      "slug": "house",
+      "difficulty": 4,
       "topics": [
       ]
     },
     {
-      "slug": "binary-search-tree",
-      "difficulty": 1,
+      "slug": "pascals-triangle",
+      "difficulty": 4,
       "topics": [
       ]
     },
     {
-      "slug": "difference-of-squares",
-      "difficulty": 1,
+      "slug": "pythagorean-triplet",
+      "difficulty": 4,
       "topics": [
       ]
     },
     {
-      "slug": "largest-series-product",
-      "difficulty": 1,
+      "slug": "saddle-points",
+      "difficulty": 4,
       "topics": [
       ]
     },
     {
-      "slug": "luhn",
-      "difficulty": 1,
+      "slug": "series",
+      "difficulty": 4,
       "topics": [
       ]
     },
     {
-      "slug": "clock",
-      "difficulty": 1,
+      "slug": "sieve",
+      "difficulty": 4,
       "topics": [
       ]
     },
     {
-      "slug": "matrix",
-      "difficulty": 1,
+      "slug": "atbash-cipher",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
-      "slug": "house",
-      "difficulty": 1,
+      "slug": "crypto-square",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
-      "slug": "zebra-puzzle",
-      "difficulty": 1,
+      "slug": "nth-prime",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
-      "slug": "minesweeper",
-      "difficulty": 1,
+      "slug": "binary-search-tree",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
-      "slug": "ocr-numbers",
-      "difficulty": 1,
+      "slug": "luhn",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
-      "slug": "wordy",
-      "difficulty": 1,
+      "slug": "ocr-numbers",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
       "slug": "food-chain",
-      "difficulty": 1,
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
-      "slug": "linked-list",
-      "difficulty": 1,
+      "slug": "custom-set",
+      "difficulty": 5,
       "topics": [
       ]
     },
     {
-      "slug": "custom-set",
-      "difficulty": 1,
+      "slug": "simple-cipher",
```

Comment:
this does have an IO monad, because of the random aspect. Thus, should we make it even higher than a 5? Compare against bank-account and linked-list both are 6.